resources.h

```diff
@@ -250,5 +279,249 @@ const static std::map<PaletteName, Palette> PALETTES({
                   { 123, 150, 249, 255 },
                   { 255, 238, 255, 255 }, 
                   { 68,  99,  51,  255 }}
+	},
+
+	{ aemil, {{0,   0,   0,   0  },
```

Comment:
This spacing needs to get cleaned up at some point, but I have a plan for that later.